spec/parameters/query/fromHeight.yml

```diff
@@ -0,0 +1,6 @@
+name: fromHeight
+in: query
+description: Filter from block height.
```

Comment:
The description should say if the block with that exact same height is included or not (inclusive or exclusive filter).

Also, the word "filter" can be a bit ambiguous. I would clarify it adding something like `Only blocks with height greater than this one are returned.`.